app/jobs/scheduled/periodical_updates.rb

```diff
@@ -48,6 +48,8 @@ def execute(args = nil)
 
       Category.auto_bump_topic!
 
+      Upload.backfill_dominant_colors!(25)
```

Comment:
Non-blocking but I noticed we reduced this from 25 -> 100, are we worried 100 might be too much because lowering this means we will only back-fill 100 per hour instead of 400 per hour. I kind of feel like since this is in a background job anyway, we can be more aggressive here.

Reply:
Problem is, when images are on S3, the server has to download them locally before it can run ImageMagick - so this can be quite time consuming depending on network conditions. The periodical_updates job needs to complete within 15 minutes, otherwise runs will start overlapping and causing issues. I think 25 is much safer.

Calculation is expedited in the post_process_cooked job when required, so it doesn't matter if this backfill takes a long time.